src/main/java/com/inventor/email/InputConsole.java

```diff
@@ -0,0 +1,92 @@
+package com.inventor.email;
+
+import org.springframework.stereotype.Component;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Component
+public class InputConsole {
+    private boolean isInput() {
+        System.out.print("Чи бажаєте ви ввести пошту? (y/n): ");
+        char answer = new Scanner(System.in).nextLine().charAt(0);
+        if(answer == 'y' || answer == 'Y'){
```

Comment:
It can be simplified to `.equalsIgnoreCase(...)`. In such case, you should consider flipping the comparison to avoid a NullPointerException